ghcup-vanilla-0.0.8.yaml

```diff
@@ -4572,6 +4572,110 @@ ghcupDownloads:
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc984-x86_64-deb11
+              dlHash: af151db8682b8c763f5a44f960f65453d794c95b60f151abc82dbdefcbe6f8ad
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-deb11
+          Linux_Fedora:
+            '>= 33':
+              dlHash: 5f03d48f118abd30aee37d8bcddc1d7012193ff205b15121807ecd979c4cf947
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc984-x86_64-ubuntu1804
+              dlHash: e826990bd46a93ff8c87cf279fe1c480d763df4f1140fdf33b26d22ef3e21f06
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc984-x86_64-ubuntu2004
+              dlHash: d83941c9be977e7dc1a6f3eca32bc11e01a7cc3e6b678a226856a99e2ae22ce9
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            '>= 8': &ghc984-x86_64-rocky8
+              dlHash: 7e10f00bdc9e552d9fc3d330b27681b340b0653df4741983b49937c2b533a5c4
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-rocky8-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc984-x86_64-ubuntu1804
+            unknown_versioning: *ghc984-x86_64-ubuntu2004
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc984-x86_64-rocky8
+          Windows:
+            unknown_versioning:
+              dlHash: 184c9184d6439f1da8bca0214011ef6074fb5946eaa5840dc0c3e884515d7c52
+              dlOutput: ghc-9.8.4-x86_64-windows.tar.xz
+              dlSubdir: ghc-9.8.4-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: e2f12a922754fd28511512875bf6d9eb3e0cce7fc963a7266f6e1661aeabd7ed
+              dlOutput: ghc-9.8.4-aarch64-darwin.tar.xz
+              dlSubdir: ghc-9.8.4-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 15cf9341437af2c6f75bbaf8222824f5b51c32f0637ed054513f165999b0a87d
+              dlOutput: ghc-9.8.4-aarch64-linux-deb10.tar.xz
```

Comment:
delete

Reply:
This disagrees with the filename of `dlUri`; is this okay?